definitions/output/all/backfill_pages.js

```diff
@@ -0,0 +1,122 @@
+const iterations = []
+const clients = constants.clients
+
+for (
```

Comment:
@rviscomi please list (if you can remember) the backfill table cases that need to be handled separately.

For example I see some non-standard crawl dates discussed in https://github.com/HTTPArchive/data-pipeline/issues/114.